eclair-core/src/main/scala/fr/acinq/eclair/remote/EclairInternalsSerializer.scala

```diff
@@ -108,7 +108,8 @@ object EclairInternalsSerializer {
       ("pingDisconnect" | bool(8)) ::
       ("maxRebroadcastDelay" | finiteDurationCodec) ::
       ("killIdleDelay" | finiteDurationCodec) ::
-      ("maxOnionMessagesPerSecond" | int32)).as[PeerConnection.Conf]
+      ("maxOnionMessagesPerSecond" | int32) ::
+      ("sendRemoteAddressInit" | bool(8))).as[PeerConnection.Conf]
```

Comment:
Should be `bool8` but apparently we have other "violations" in that file.

Reply:
I saw that, that's why I chose to be consistent instead of using `bool8`. I'll do a follow-up tiny refactoring to move to `bool8` everywhere in another PR.